GoogleForCreators/web-stories-wp
# 3P Media: Fix media provider attribution pills

## Context

Attributions were not clickable and the Coverr logo was missing colors

## Summary



* Increases `z-index` to ensure the media3p provider attribution pill is clickable again.
* Moves the Coverr logo SVG to an `images` folder, as SVGs in such a folder will preserve colors.

## Relevant Technical Choices



N/A

## To-do



N/A

## User-facing changes



![Screenshot 2021-06-23 at 17 07 37](https://user-images.githubusercontent.com/841956/123122032-f6993600-d445-11eb-92e8-cf11f7cfafb4.png)


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Go to third-party media tab
2. Ensure all provider attribution pills in the bottom are clickable and open in a new tab
3. Ensure that the Coverr logo has colors

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8062
Fixes #8063
